src/styles/blocks/contact.scss

```diff
@@ -0,0 +1,36 @@
+.contact {
+  &__info {
+    font-weight: 400;
+    font-size: 14px;
+    margin-bottom: 24px;
+
+    &:last-child {
+      margin-bottom: 0;
+    }
+
+    &-title {
+      margin-bottom: 8px;
+      color: #9fadc2;
+    }
+
+    &-link {
+      color: #3e3e3e;
+    }
```

Comment:
Why you wrote class-names like this(&-)?

More about naming convention in BEM, and why we use '-' in names watch [in this video](https://mate.academy/learn/css-advanced/bem#/video/920)